label_studio/projects/models.py

```diff
@@ -84,6 +84,13 @@ def with_counts_annotate(queryset):
 
 
 class Project(ProjectMixin, models.Model):
+    class SkipQueue(models.TextChoices):
```

Comment:
Do we need to put this class here?

Reply:
It's for more handy usage via `project.SkipQueue.REQUEUE_FOR_ME` 

Example from django docs: https://docs.djangoproject.com/en/3.0/ref/models/fields/#enumeration-types